### PR TITLE
Created buzzer module header file

### PR DESCRIPTION
**Pull request by nikolakosaric**
Still need to develop the .c file

### CODE REVIEW

File: include/buzzer.h [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#include <stdint.h>
+#include "hal_general.h"
```
Comment (muhlbaier) [inline include/buzzer.h]: You don't need to include hal_general.h here and you certainly don't need it twice

File: include/buzzer.h [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ 
+void hal_Start_Buzzer(uint16_t frequency);
+
+ /** @brief hardware abstraction layer to stop buzzer
+ * 
+ */
+ 
+void hal_Stop_Buzzer(void);
+ 
+
```
Comment (muhlbaier) [inline include/buzzer.h]: missing @} for doxygen

File: include/buzzer.h [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+ * @{
+ */
+
+/** @brief initializes the buzzer module
+ *
+ */
+
+void Buzzer_Init(void);
+
+/** @brief
```
Comment (muhlbaier) [inline include/buzzer.h:36]: Add brief documentation

File: include/buzzer.h [hunk 4]
```diff
@@ -0,0 +1,50 @@
+#include <stdint.h>
+#include "hal_general.h"
+#include "hal_general.h"
+
+
+#ifndef _BUZZER_H_
+#define _BUZZER_H_
+
+/**
+ * @defgroup buzzer_module Buzzer Module
```
Comment (muhlbaier) [inline include/buzzer.h:10]: expand the module documentation and include a list of dependent modules (i.e. Timing and Task)

**Review by muhlbaier — CHANGES_REQUESTED** [review]

**Comment by nikolakosaric** [comment]
I've revised the header file based on the comments made above. As the module is developed, the documentation will be referenced and corrected if need be

**Review by muhlbaier — APPROVED** [review]
Looks good.

**Pull request merged**